src/components/LinkEditor.tsx

```diff
@@ -155,25 +182,41 @@ class LinkEditor extends React.Component<Props, State> {
   };
 
   handleOpenLink = (event): void => {
-    const { href } = this.props.mark.attrs;
-
     event.preventDefault();
-    this.props.onClickLink(href);
+    this.props.onClickLink(this.href);
+  };
+
+  handleCreateLink = (value: string) => {
+    this.discardInputValue = true;
+    const { onCreateLink } = this.props;
+
+    if (onCreateLink) return onCreateLink(value);
```

Comment:
I need to add a eslint rule to enforce this

```suggestion
    if (onCreateLink) {
      return onCreateLink(value);
    }
```

Reply:
Probably not, I think a guard that just returns is more justified – don't worry about it, I'll just implement the eslint rule and automate the formatting so it's not a discussion 👍